ironfish/src/rpc/routes/wallet/getAssets.ts

```diff
@@ -38,6 +40,10 @@ export const GetAssetsResponseSchema: yup.ObjectSchema<GetAssetsResponse> = yup
     owner: yup.string().defined(),
     status: yup.string().defined(),
     supply: yup.string().optional(),
+    verification: yup
```

Comment:
I think this should be `yup.object({ status: yup.string() }).defined()` instead of using shape() which is used for dynamic types.